src/libraries/Microsoft.PowerFx.Connectors/ConnectorFunction.cs

```diff
@@ -615,6 +615,12 @@ internal async Task<FormulaValue> InvokeInternalAsync(FormulaValue[] arguments,
                 throw new InvalidOperationException($"In namespace {Namespace}, function {Name} is not supported.");
             }
 
+            FormulaValue ev = arguments.Where(arg => arg is ErrorValue).FirstOrDefault();
```

Comment:
>FormulaValue ev = arguments.Where(arg => arg is ErrorValue).FirstOrDefault(); [](http://example.com/codeflow?start=12&length=77)

Impressively, we were not propagating errors when an Error was present in arguments